### PR TITLE
[D2IQ-63899] docs for elastic 3.1.0-7.4.1 upgrade

### PR DESCRIPTION
**Pull request by viivek46**
## Jira Ticket
https://jira.mesosphere.com/browse/D2IQ-63899

## Description of changes being made
Docs for elastic upgrade to 3.1.0-7.4.1

### CODE REVIEW

File: pages/mesosphere/dcos/services/elastic/3.1.0-7.4.1/release-notes/index.md [hunk 1]
```diff
@@ -8,6 +8,26 @@ model: /mesosphere/dcos/services/elastic/data.yml
 render: mustache
 ---
 
+# Release notes for Elastic Service version 3.1.0-7.4.1
+
+## Upgrades
+
+-   Upgraded Elasticsearch and Kibana to [7.4.1](https://www.elastic.co/guide/en/elasticsearch/reference/7.4/release-notes-7.4.1.html)
+-   Upgraded SDK version from `0.56.3` to `0.57.3`. ([#70](https://github.com/mesosphere/dcos-elastic-service/pull/70))
```
Comment (kaiwalyajoshi) [inline pages/mesosphere/dcos/services/elastic/3.1.0-7.4.1/release-notes/index.md]: Given the bug fix in SDK `0.57.3` which fixes the possiblity of a data-loss, can you add the `pod replace` notice which is found in the SDK release notes here.

**Comment by viivek46** [comment]
> https://www.elastic.co/guide/en/elastic

Thanks @rishabh96b ,
Nice catch, will update those links. seems like many are missed in previous releases!

File: pages/mesosphere/dcos/services/elastic/3.1.0-7.4.1/release-notes/index.md [hunk 2]
```diff
@@ -8,6 +8,26 @@ model: /mesosphere/dcos/services/elastic/data.yml
...
+    -   [0.57.1](https://github.com/mesosphere/dcos-commons/releases/tag/0.57.1)
+    -   [0.57.2](https://github.com/mesosphere/dcos-commons/releases/tag/0.57.2)
+    -   [0.57.3](https://github.com/mesosphere/dcos-commons/releases/tag/0.57.3)
+    
+## Bug Fixes
+
+-   Fixed Kibana Dashboard issue. ([#72](https://github.com/mesosphere/dcos-elastic-service/pull/72)
+
+
 # Release notes for Elastic Service version 3.0.0-7.3.2
```
Comment (SarahStegall) [inline pages/mesosphere/dcos/services/elastic/3.1.0-7.4.1/release-notes/index.md]: Why are we including release notes for a version which we do not offer or support?

Comment (viivek46) [reply pages/mesosphere/dcos/services/elastic/3.1.0-7.4.1/release-notes/index.md]: Hi Sarah, 
We Does support this version as well but for an information we are keeping just previous version as well in release notes!

Comment (SarahStegall) [reply pages/mesosphere/dcos/services/elastic/3.1.0-7.4.1/release-notes/index.md]: I have added a note to the Release Notes: "Although version 3.0.0-7.3.2 is no longer supported, the following changes were implemented in it. You will need to be aware of these changes when you upgrade to the current version, 3.1.0-7.4.1."

I also added a link to the Updates page from the Release Notes.

**Comment by viivek46** [comment]
@cmastr This PR is ready to merge!

**Pull request merged**